rpc/wrtc_client_stream.go

```diff
@@ -194,8 +212,27 @@ func init() {
 }
 
 func (s *webrtcClientStream) writeMessage(m interface{}, eos bool) (err error) {
+	s.mu.RLock()
```

Comment:
`writeMessage` is safe for concurrent use now. for multiple writers non EOS it grabs a read lock. for an EOS, it grabs a write lock